src/MooLite/plugins/EnhancerHelper/EnhancerHelperPlugin.ts

```diff
@@ -0,0 +1,213 @@
+import { MooLitePlugin } from "src/MooLite/core/plugins/MooLitePlugin";
+import { MooLiteTab } from "src/MooLite/core/plugins/MooLiteTab";
+import { markRaw } from "vue";
+import EnhancerHelperPluginDisplay from "src/MooLite/plugins/EnhancerHelper/EnhancerHelperPluginDisplay.vue";
+import { ItemDetail } from "src/MooLite/core/inventory/items/ItemDetail";
+import { ItemAmount } from "src/MooLite/core/inventory/items/ItemAmount";
+import { ItemHrid } from "src/MooLite/core/inventory/ItemHrid";
+import { CharacterItem } from "src/MooLite/core/inventory/CharacterItem";
+
+export class EnhancerHelperPlugin extends MooLitePlugin {
+    name: string = "Enhancer Helper";
+    key = "enhancer-helper";
+    description: string = "Various enhancing tools - By Void";
+
+    tab: MooLiteTab = {
+        icon: "💎",
+        pluginName: this.name,
+        componentName: "EnhancerHelperPluginDisplay",
+        component: markRaw(EnhancerHelperPluginDisplay),
+    };
+
+    getEnhanceableItems(): ItemDetail[] {
+        return this._game.inventory.sortedAlphabeticalItems.filter((item) => item.enhancementCosts != null);
+    }
+
+    getEnhancementMaterials(itemHrid: ItemHrid): ItemAmount[] | undefined | null {
+        return this._game.inventory.sortedAlphabeticalItems.find((item) => item.hrid === itemHrid)?.enhancementCosts;
+    }
+
+    getItemName(itemHrid: ItemHrid): String | null {
+        return this._game.inventory.itemDetailMap[itemHrid].name;
+    }
+
+    getEnhancingLevel(): number {
+        return this._game.skills.getLevel("/skills/enhancing");
+    }
+
+    getEnhancementSuccessTable(enhancingLevel: number, itemLevel: number, toolBonus: number): number[] {
+        let enhancementLevelBonus = 0;
+        const enhancingTea = <HTMLInputElement>document.getElementById("enhancingTea");
+        const superEnhancingTea = <HTMLInputElement>document.getElementById("superEnhancingTea");
```

Comment:
Do not query for html but use Vue form input bindings and pass as arguments
https://vuejs.org/guide/essentials/forms.html